tests/test_operations.py

```diff
@@ -78,4 +98,65 @@ def test_update(self, graph_data_update, requests_mock):
         """Test update function"""
         requests_mock.put(data_url)
         r = operations.update(data_url, data=graph_data_update)
-        assert r is None
\ No newline at end of file
+        assert r is None
+
+class TestException:
+    """Test exceptions and errors for fdpclient.operations functions"""
+
+    # test unexpected errors
+    def test_create_unexpected_error(self, data, requests_mock, capsys):
```

Comment:
What does this actually test? That if requests raises an exception it is not handled but passed to the caller?

Reply:
yes, the requests exceptions will be passed to the caller and then re-raise with a bit more info about the caller